File: sample-app/src/debug/java/org/granchi/whatnow/framework/dagger/DebugWhatNowModule.java
package org.granchi.whatnow.framework.dagger;

import org.granchi.whatnow.DebugWhatNowApplication;

import dagger.Module;

/**
 * Dagger module for the debug build variant of the app.
 *
 * @author serandel
 */
@Module
public class DebugWhatNowModule extends WhatNowModule {
    public DebugWhatNowModule(DebugWhatNowApplication app) {
        super(app);
    }
}

File: sample-app/src/debug/kotlin/org/granchi/whatnow/framework/dagger/DebugWhatNowModule.kt
package org.granchi.whatnow.framework.dagger

import dagger.Module
import org.granchi.whatnow.DebugWhatNowApplication

/**
 * Dagger module for the debug build variant of the app.
 */
@Module
class DebugWhatNowModule(app: DebugWhatNowApplication) : WhatNowModule(app)
